packages/contracts/src/dollar/core/UbiquityDollarManager.sol

```diff
@@ -81,7 +79,7 @@ contract UbiquityDollarManager is AccessControl {
         // to be removed
 
         TWAPOracleDollar3pool oracle = TWAPOracleDollar3pool(twapOracleAddress);
-        oracle.update();
+        ///oracle.update();
```

Comment:
```suggestion

```

We shouldn't update the oracle here?

Reply:
On local testnet, it fails unless a trade has taken place, like the `metapool.get_price_cumulative_last()` in the constructor. I don't understand why the behaviour works on mainnet but not a local testnet.

Reply:
The values are also null until a trade takes place so we're not losing anything by not calling it here.

Reply:
@zgorizzo69 do you have any remarks on this?